2018AsianGamesApp/app/src/main/java/com/example/mrtbkwjn/a2018asiangamesapp/LoginActivity.java

```diff
@@ -0,0 +1,71 @@
+package com.example.mrtbkwjn.a2018asiangamesapp;
+
+import android.app.ActionBar;
+import android.content.Intent;
+import android.os.Build;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.view.WindowManager;
+import android.widget.Button;
+import android.widget.ImageButton;
+
+public class LoginActivity extends AppCompatActivity {
+
+    private ImageButton googleButton = (ImageButton) findViewById(R.id.glogButton);
+    private ImageButton facebookButton = (ImageButton) findViewById(R.id.fblogButton);
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+
+        //View decorView = getWindow().getDecorView();
```

Comment:
Consider deleting these commented codes later on the final submissions. Commented codes makes it harder to read the codes later